testing/simulator/Cargo.toml

```diff
@@ -7,13 +7,10 @@ edition = { workspace = true }
 
 [dependencies]
 clap = { workspace = true }
-env_logger = { workspace = true }
 environment = { workspace = true }
-eth2_network_config = { workspace = true }
 execution_layer = { workspace = true }
 futures = { workspace = true }
 kzg = { workspace = true }
-logging = { workspace = true }
```

Comment:
@ackintosh it turns out we're still using the `logging` crate for `build_workspace_filter`, I think we add this back?